server/pulp/server/db/reaper.py

```diff
@@ -45,8 +46,8 @@ def reap_expired_documents():
     number of days as the argument.
     """
     _logger.info(_('The reaper task is cleaning out old documents from the database.'))
-    for model, config_name in _COLLECTION_TIMEDELTAS.items():
+    for _model, config_name in _COLLECTION_TIMEDELTAS.items():
```

Comment:
using _model makes me think it is a private module level variable.  How about model_class as an alternative?